src/components/table/src/_template.njk

```diff
@@ -0,0 +1,38 @@
+{% if params.scrollable %}
```

Comment:
All of the hardcoded aria-labels etc should probably be parameterised to allow for internationalisation